src/Phinx/Db/Adapter/BambooHRMysqlAdapter.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Phinx\Db\Adapter;
+
+class BambooHRMysqlAdapter extends \Phinx\Db\Adapter\MysqlAdapter {
+
+	public function hasTable($tableName) {
+		$sql = sprintf("DESC %s", $tableName);
+		try {
+			$exists = $this->query($sql);
```

Comment:
$exists is not used anywhere. You might be able to remove the reference and just execute the query.